src/EventGridExtension/EventGridExtensionConfig.cs

```diff
@@ -43,6 +58,24 @@ public void Initialize(ExtensionConfigContext context)
                 .AddOpenConverter<JObject, OpenType.Poco>(typeof(JObjectToPocoConverter<>))
                 .BindToTrigger<JObject>(new EventGridTriggerAttributeBindingProvider(this));
 
+            // Register the output binding
+            var rule = context
+                .AddBindingRule<EventGridAttribute>();
+            rule.BindToCollector(_converter);
```

Comment:
The trigger binding supports EventGridEvent, JObject and string as parameter types, via converters registered. Does your output binding also support this?

Reply:
Test to verify, but it should be the case that if you have an implementation for IAsyncCollector<EventGridEvent> and a valid chain of conversions from each of the supported types to EventGridEvent, the system will convert for you.